content/topics/Analyze/causal-inference/introduction/synthetic-control-methods-intro.md

```diff
@@ -0,0 +1,268 @@
+- No Extrapolation: In contrast to some regression-based methods that might predict beyond the range of observed data, SCM guarantees that the constructed synthetic control remains within the actual observed outcomes of the control units.
+- Safeguard against specification searches and p-hacking: The weights for synthetic controls can be determined and pre-registered before the post-treatment outcomes are known or before the intervention occurs.
+
+
+Possible disadvantages include: 
+
+- SCMs require extensive data on pre-treatment characteristics and outcomes for both treated and potential control units. 
+- The results of SCMs can be sensitive to the choice of predictor variables and the weighting scheme.
+There can be considerable variability in estimates depending on the specification of the synthetic control model.
+- Constructing synthetic controls can be computationally intensive, especially with large datasets or complex models.
+- Bias in small samples: When there are short pre-treatment periods, large donor pools, and significant noise, there is an increased risk of interpolation bias. This means the synthetic control may be overly tailored to the specific characteristics of the pre-treatment period, leading to overfitting.
+- Inference: Traditional methods like t-tests and p-values are not well-suited for SCM. Instead, permutation tests are used, which have their own limitations, especially with small donor pools and the assumption that any control unit could have been treated. This assumption is problematic if control units differ significantly from each other or from the treated unit. Additionally, this type of inference is computationally intensive.
+
+## Summary 
+
+{{% summary %}}
+
+- Synthetic control methods (SCMs) are statistical techniques used to estimate the effect of interventions by creating a synthetic version of the treatment group from a weighted combination of control units.
+- SCMs provide a transparent, data-driven approach for constructing control groups, making them suitable for contexts where randomized controlled trials are not feasible.
+- While SCMs offer flexibility and systematic estimation, they require extensive pre-treatment data and can be sensitive to model specifications and computationally intensive.
+
+{{% /summary %}}
+
+
+### References 
+
+- Abadie, Alberto. 2021. [Using Synthetic Controls: Feasibility, Data Requirements, and Methodological Aspects]( https://www.aeaweb.org/articles?id=10.1257/jel.20191450) Journal of Economic Literature, 59 (2): 391-425.
+
+- Alberto Abadie, Alexis Diamond & Jens Hainmueller (2010) Synthetic
+[Control Methods for Comparative Case Studies: Estimating the Effect of California’s Tobacco
+Control Program](https://www.tandfonline.com/doi/abs/10.1198/jasa.2009.ap08746), Journal of the American Statistical Association, 105:490, 493-505, DOI:
+10.1198/jasa.2009.ap08746
+
+- [Code snippets](https://github.com/edunford/tidysynth/blob/master/README.md)
```

Comment:
rename as [Tidysynth: A Tidy Implementation of Synthetic Control Methods]( add link)